File: src/targets/java/asynchttp/fixtures/multipart-form-data-no-params.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
<<<<<<< HEAD
client.prepare("POST", "https://httpbin.org/anything")
=======
client.prepare("POST", "http://mockbin.com/har")
>>>>>>> upstream/master
  .setHeader("Content-Type", "multipart/form-data")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: src/targets/ocaml/cohttp/fixtures/multipart-form-data-no-params.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

<<<<<<< HEAD
let uri = Uri.of_string "https://httpbin.org/anything" in
=======
let uri = Uri.of_string "http://mockbin.com/har" in
>>>>>>> upstream/master
let headers = Header.add (Header.init ()) "Content-Type" "multipart/form-data" in

Client.call ~headers `POST uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)